Detail/ObjectView.swift

```diff
@@ -76,6 +81,15 @@ class ObjectView: UIView {
                 listComponentView.translatesAutoresizingMaskIntoConstraints = false
                 contentView.addSubview(listComponentView)
                 componentView = listComponentView
+            case .custom:
+                if let listComponentView = dataSource.customComponentView(for: component, in: self) {
+                    listComponentView.translatesAutoresizingMaskIntoConstraints = false
+                    contentView.addSubview(listComponentView)
+                    componentView = listComponentView
+                } else {
+                    print("Custom component type at index \(index) ignored")
```

Comment:
I think it's safe to remove the entire else statement here

Reply:
No, because then the `componentView` is never initialized. I get errors when the layouts are activated if I don't have a case for handling custom views being `nil`.